### ISSUE TITLE
Sudden unexpected death in epilepsy

### ISSUE THREAD

**Issue by kearneyja** [obophenotype/human-phenotype-ontology#6210]
**Preferred term label:**
Sudden unexpected death in epilepsy

**Synonyms**
SUDEP
sudden unexplained death in epilepsy

**Definition (free text, please give PubMed ID)**
Sudden, unexpected death in a person with epilepsy, with or without evidence for a seizure preceding the death, in which there is no evidence of other disease, injury, or drowning that caused the death.
PMID: 22191982

**Parent term (use hpo.jax.org/app)**
HP0001250 Epilepsy

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0003-1726-4769

**Comment by pnrobinson** [comment]
The HPO does not code things like the assumed etiology. We would simply use the term for "Sudden death"  https://hpo.jax.org/app/browse/term/HP:0001699. If you were doing an analysis, then you would encode the status of individuals as being affected by epilepsy or not somewhere else.
